From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Otto=20Pfl=C3=BCger?= <otto.pflueger@abscue.de>
Date: Sat, 17 May 2025 20:32:36 +0200
Subject: [PATCH 104/225] dt-bindings: spi: sprd: add compatibles for UMS9230
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Add compatible strings for the SPI and ADI controllers on UMS9230.

Signed-off-by: Otto Pflüger <otto.pflueger@abscue.de>
---
 .../devicetree/bindings/spi/sprd,sc9860-spi.yaml    |  7 ++++++-
 .../devicetree/bindings/spi/sprd,spi-adi.yaml       | 13 +++++++++----
 2 files changed, 15 insertions(+), 5 deletions(-)

diff --git a/Documentation/devicetree/bindings/spi/sprd,sc9860-spi.yaml b/Documentation/devicetree/bindings/spi/sprd,sc9860-spi.yaml
index d55c01e9a..2f3e308ae 100644
--- a/Documentation/devicetree/bindings/spi/sprd,sc9860-spi.yaml
+++ b/Documentation/devicetree/bindings/spi/sprd,sc9860-spi.yaml
@@ -13,7 +13,12 @@ maintainers:
 
 properties:
   compatible:
-    const: sprd,sc9860-spi
+    oneOf:
+      - items:
+          - enum:
+              - sprd,ums9230-spi
+          - const: sprd,sc9860-spi
+      - const: sprd,sc9860-spi
 
   reg:
     maxItems: 1
diff --git a/Documentation/devicetree/bindings/spi/sprd,spi-adi.yaml b/Documentation/devicetree/bindings/spi/sprd,spi-adi.yaml
index 903b06f88..333bf674c 100644
--- a/Documentation/devicetree/bindings/spi/sprd,spi-adi.yaml
+++ b/Documentation/devicetree/bindings/spi/sprd,spi-adi.yaml
@@ -47,10 +47,15 @@ allOf:
 
 properties:
   compatible:
-    enum:
-      - sprd,sc9860-adi
-      - sprd,sc9863-adi
-      - sprd,ums512-adi
+    oneOf:
+      - items:
+        - enum:
+            - sprd,ums9230-adi
+        - const: sprd,ums512-adi
+      - enum:
+        - sprd,sc9860-adi
+        - sprd,sc9863-adi
+        - sprd,ums512-adi
 
   reg:
     maxItems: 1
